about.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="./images/favicon.ico" type="image/x-icon" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="about.css" />
+    <title>About</title>
+  </head>
+  <body>
+    <header>
+      <div id="first-header">
+        <img src="./images/logo-facebook.png" alt="" />
+        <img src="./images/logo-tweeter.png" alt="" />
+        <ul id="navbar-h1">
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+    </header>
+    <div id="second-header">
+      <a href="./index.html" id="logo-header"
+        ><img src="./images/logo.png" alt=""
+      /></a>
+      <ul id="navbar-h2">
+        <li><a href="./about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li><button>CC Campaign</button></li>
+      </ul>
+    </div>
+    <section id="section1">
+      <nav class="menu-responsive">
+        <div class="x-section">
+          <img src="./images/x-icon.png" alt="" />
+        </div>
+        <ul class="navbar-menu-responsive">
+          <li><a href="./index.html" class="items-menu-responsive">Home</a></li>
+          <li><a href="#" class="items-menu-responsive">Program</a></li>
+          <li><a href="#" class="items-menu-responsive">Join</a></li>
+          <li><a href="#" class="items-menu-responsive">Sponsor</a></li>
+          <li><a href="#" class="items-menu-responsive">News</a></li>
+          <li><button>CC Campaign</button></li>
+        </ul>
+      </nav>
+      <div id="main-banner">
+        <img id="hamburg" src="./images/menu.png" alt="" />

```

Comment:
- [ ] Your Hamburger menu would be better placed in your `<header>` tag since it is independent of any sections. Please consider moving it there! 🤓 🔧 

⚠️ **Consider applying these changes to your `index.html` too!** ⚠️

Reply:
@anagudelogu  I understand your correction, semantically I also thought the same thing, but I need the hamburger logo to be located there because if you see the Behance design it is in the same "main-banner" section. I want it to be behind the hamburger the same main-banner background because the design requires it. I can't put it in the headers either because they disappear in the mobile version, therefore the hamburger would also disappear, you know? I agree with you, regarding the semantics, but in this design, I cannot apply it. I'm very sorry!

Reply:
Hi, @rodrigopenayo98 Thanks for reaching out! What you mention is not necessarily true, you can achieve the same result using CSS. Markup and styles are very different! You can use `position: absolute` to remove the header from the normal document flow (causing it to overlap the main-banner background) and style it with a transparent background and a hamburger icon to the left. Remember that although every HTML element has default styles, you can modify them in order to achieve what you need. Hope this helps you while finding a solution! 🤓 🔧

Reply:
@anagudelogu Yeah! you're right! I solved it thanks!